packages/types/src/block-kit/composition-objects.ts

```diff
@@ -1,5 +1,14 @@
 // This file contains objects documented here: https://api.slack.com/reference/block-kit/composition-objects
 
+/**
+ * Re-usable labels for common color schemes present in Slack. `danger` displays with a red background (red text on
+ * mobile), while `primary` displays with a green background (green text on mobile).
+ */
+export type ColorScheme = 'primary' | 'danger';
+
+/** The conversation type as available within the Slack UI. */
+export type Conversation = 'im' | 'mpim' | 'private' | 'public';
```

Comment:
How about renaming this to ConversationType or something like that to make it clearer? Once we export it, renaming it later will be a challenge.